lib/graphql/introspection/input_value_type.rb

```diff
@@ -8,8 +8,12 @@
   field :type, !GraphQL::Introspection::TypeType
   field :defaultValue, types.String, "A GraphQL-formatted string representing the default value for this input value." do
     resolve ->(obj, args, ctx) {
-      value = obj.default_value
-      value.nil? ? nil : JSON.generate(obj.type.coerce_result(value), quirks_mode: true)
+      if obj.default_value?
+        value = obj.default_value
+        value.nil? ? 'null' : JSON.generate(obj.type.coerce_result(value), quirks_mode: true)
```

Comment:
:+1: sounds better to leave it as is. It _might work_ to send it through coerce_result but you end up printing something other than the user-provided value!!